player/main.c

```diff
@@ -144,6 +152,7 @@ void mp_print_version(struct mp_log *log, int always)
     int v = always ? MSGL_INFO : MSGL_V;
     mp_msg(log, v, "%s %s\n built on %s\n",
            mpv_version, mpv_copyright, mpv_builddate);
+    mp_msg(log, v, "libplacebo version: %s\n", MP_PL_VERSION);
```

Comment:
my bike shed contribution: I'd rather have the line be missing

Reply:
And I tried to have the `#if` in a single location ^^; which is not in the middle of the code.